packages/account-v2/src/modules/AddressFields/validations.ts

```diff
@@ -0,0 +1,66 @@
+import * as Yup from 'yup';
+
+const regexChecks = {
+    address_details: {
+        address_city: /^\p{L}[\p{L}\s'.-]{0,99}$/u,
+        address_line_1: /^[\p{L}\p{Nd}\s'.,:;()\u00b0@#/-]{1,70}$/u,
+        address_line_2: /^[\p{L}\p{Nd}\s'.,:;()\u00b0@#/-]{0,70}$/u,
+        address_postcode: /^[a-zA-Z0-9\s-]{0,20}$/,
+        address_state: /^[\w\s\W'.;,-]{0,99}$/,
+        non_jersey_postcode: /^(?!\s*je.*)[a-zA-Z0-9\s-]*/i,
+    },
+};
+
+const addressPermittedSpecialCharactersMessage = ". , ' : ; ( ) ° @ # / -";
+
+export const addressDetailValidations = (countryCode: string, isSvg: boolean) => ({
+    addressCity: Yup.string()
+        .required('City is required')
+        .max(99, 'Only 99 characters, please.')
+        .matches(
+            regexChecks.address_details.address_city,
+            'Only letters, periods, hyphens, apostrophes, and spaces, please.'
```

Comment:
feels like we can make this message into an object as well 🤔 cause there's a few places where we use the same message

Reply:
For now this is only being used here, we can come back to it if we need it to be in a const.